tools/liveimage-mount

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/python -tt
+# coding: latin-1
+# 2015-07-23 13:03:05
```

Comment:
`# coding: latin-1` is used by Python to interpret the ® symbol in line 6. See https://www.python.org/dev/peps/pep-0263/


`# 2015-07-23 13:03:05` is the first feature from the commit statement, "Include a datetime tag to easily distinguish in-the-wild versions."   Since this program can run stand-alone and be distributed without the rest of the livecd-tools package, this is a simple way to distinguish among different versions that a client may be working with.